api/localdev.env

```diff
@@ -1,3 +1,6 @@
+# Switch AUTH Providers ['KEYCLOAK','AWS_COGNITO']
+export AUTH_PROVIDER='AWS_COGNITO' # FOM is targeting to use AWS Cognito (FAM) in the future.
+
 # Environment variable settings needed to run API locally via npm run start:api
 export KEYCLOAK_ENABLED='false'
```

Comment:
Don't we want to change this variable, maybe something like SECURITY_ENABLED or OIDC_ENABLED, I don't know.

Reply:
A similar one for Cognito is also here in the file: "aws-cognito-env.json". But I will see if it really make sense to unite this variable into what you mention for both keycloak and aws without impacting too much change. Thanks for point that out.

Reply:
That's a good suggestion. I updated it to "SECURITY_ENABLED".